test/e2e/api/matchMedia/map-projection.html

```diff
@@ -0,0 +1,93 @@
+                osmtile = map.querySelector('#OSMTILE'),
+                cbmtile = map.querySelector('#CBMTILE');
+
+            const hideLayer = (matchesOSMTILE) => {
+                map.whenLayersReady().then(() => {
+                    if (matchesOSMTILE) {
+                        cbmtile.setAttribute('hidden', 'hidden');
+                        osmtile.removeAttribute('hidden');
+                    } else {
+                        osmtile.setAttribute('hidden', 'hidden');
+                        cbmtile.removeAttribute('hidden');
+                    }
+                });
+            };
+            hideLayer(projectionMatcher.matches);
+            document.querySelector('.switchOSM').addEventListener('click', () => map.projection = 'OSMTILE');
+            document.querySelector('.switchCBMT').addEventListener('click', () => map.projection = 'CBMTILE');
+
+            const changeHandler = (e) => {
+                let matchesOSMTILE = projectionMatcher.matches;
+                hideLayer(matchesOSMTILE);
+            };
+            projectionMatcher.addEventListener('change', changeHandler);
+        });
+    </script>
+
+    <mapml-viewer projection="OSMTILE" zoom="9" lat="44.630550504861795" lon="-103.86611938476564" controls>
+        <map-layer id="OSMTILE" label="OpenStreetMap" checked="">
+            <map-link rel="license" title="Â© OpenStreetMap contributors CC BY-SA"
+                href="https://www.openstreetmap.org/copyright"></map-link>
+            <map-extent units="OSMTILE" checked="checked">
+                <map-input name="z" type="zoom" value="18" min="0" max="18"></map-input>
+                <map-input name="x" type="location" units="tilematrix" axis="column" min="0" max="262144"></map-input>
+                <map-input name="y" type="location" units="tilematrix" axis="row" min="0" max="262144"></map-input>
+                <map-link rel="tile" tref="https://tile.openstreetmap.org/{z}/{x}/{y}.png"></map-link>
+            </map-extent>
+        </map-layer>
+        <map-layer media="(map-projection: CBMTILE)" id="CBMTILE" checked>
+            <map-extent label="CBMT - Lambert" units="CBMTILE" checked>
+                <map-input name="z" type="zoom" min="0" max="18"></map-input>
+                <map-input name="y" type="location" units="tilematrix" axis="row"></map-input>
+                <map-input name="x" type="location" units="tilematrix" axis="column"></map-input>
+                <map-link rel="tile"
+                    tref="https://maps-cartes.services.geo.ca/server2_serveur2/rest/services/BaseMaps/CBMT3978/MapServer/tile/{z}/{y}/{x}?m4h=t"></map-link>
+            </map-extent>
+        </map-layer>
+    </mapml-viewer>
+
+    <div class="description">
+        <h2>Descriptions</h2>
+        <button class="switchOSM">Change projection to OSMTILE</button>
+        <button class="switchCBMT">Change projection to CBMTILE</button>
+    </div>
+</body>
+
+</html>
```

Comment:
If you ever wonder why this icon appears in some github listings: 
![image](https://github.com/user-attachments/assets/73f9d0df-94c1-43a4-a796-144e02cdc336)
it's because git wants a blank line at the end of your file. IIRC it's something to do with file diffs, but I don't know if it makes any difference in practice.